.github/workflows/docker.yml

```diff
@@ -0,0 +1,88 @@
+name: Docker
+
+on:
+  workflow_call:
+    secrets:
+      DOCKERHUB_TOKEN:
+        required: true
+      DOCKERHUB_USERNAME:
+        required: true
+
+jobs:
+  docker-meta:
+    runs-on: ubuntu-latest
+    outputs:
+      labels: ${{ steps.meta-main.outputs.labels || steps.meta-version.outputs.labels }}
+      tags: ${{ steps.meta-main.outputs.tags || steps.meta-version.outputs.tags }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Docker meta
+        id: meta-main
+        uses: docker/metadata-action@v4
+        if: startsWith(github.ref, 'refs/tags/v') || (github.ref == 'refs/heads/main')
+        with:
+          images: |
+            solidproject/community-server
+          # edge will always be executed (without latest tag), semver only on tag push events (with latest tag)
+          tags: |
+            type=edge
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}
+          github-token: ${{ secrets.github_token }}
+      - name: Docker meta
```

Comment:
This step has the same name as the previous step. Is this intended?

Reply:
yes they perform similar tasks just with different conditions. But I've changed the names to better reflect this.